Bottom: be8d75f7d9372464eb70c8c89bc52f63b7a321fa
Top:    ad08e00a8f438f1a1a205c62d392402bf1470a18
Author: Ian Kent <ikent@redhat.com>
Date:   2013-10-17 10:20:13 +0800

autofs-5.0.7 - fix automount(8) mailing list info

Replace a reference to the old mailman mailing list in automount(8).


---

diff --git a/man/automount.8 b/man/automount.8
index bfcb41e..79e53f0 100644
--- a/man/automount.8
+++ b/man/automount.8
@@ -181,7 +181,7 @@ Don't know, I've fixed everything I know about.
 The documentation could be better.
 
 Please report other bugs along with a detailed description to
-<autofs@linux.kernel.org>. For instructions on how to join the list
-and for archives visit http://linux.kernel.org/mailman/listinfo/autofs
+<autofs@vger.kernel.org>. Visit http://vger.kernel.org/vger-lists.html#autofs
+for information about the list.
 .SH AUTHOR
 H. Peter Anvin <hpa@transmeta.com> and Ian Kent <raven@themaw.net>.
